packages/flutter/test/widgets/focus_manager_test.dart

```diff
@@ -1,6 +1,7 @@
 // Copyright 2014 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+import 'dart:io';
```

Comment:
I believe there should be a blank line above this one to make the license checker happy.

Reply:
Done. Thanks.

Reply:
Now the analyzer is complaining that this import is not needed...